foundry/applications/formApplications/baseEntitySheets/rollTableConfig.d.ts

```diff
@@ -1,25 +1,151 @@
-// @TODO:
-
 /**
  * The RollTable configuration sheet
- *
- * @param table - The rollable table entity being configured
- * @param options - Additional application rendering options
  */
-declare class RollTableConfig extends BaseEntitySheet {
+declare class RollTableConfig extends BaseEntitySheet<RollTableConfig.Data, RollTable> {
+  static get defaultOptions(): RollTableConfig.Options;
+
   /**
-   * Return a dynamic application title for the configuration sheet
+   * @override
    */
   get title(): string;
 
   /**
-   * Activate event listeners for the RollTable Configuration Sheet
+   * @param options - (unused)
+   * @override
+   */
+  getData(options?: Application.RenderOptions): RollTableConfig.Data;
+
+  /**
+   * @override
    */
   activateListeners(html: JQuery): void;
 
   /**
-   * @param options - (unused)
+   * Handle creating a TableResult in the RollTable entity
+   * @param event      - The originating mouse event
+   * @param resultData - An optional object of result data to use
+   */
+  protected _onCreateResult(event: MouseEvent, resultData?: object): Promise<any>; // TODO: type when TableResult is typed
```

Comment:
Regarding all those TODOs: `RollTable` might not have been updated for it yet but I think the type that all references to "TableResult" embedded entities should have is just `RollTable.Result`. I checked and this actually seems to be typed correctly. there is no class `TableResult`, it always is just the data (that's probably changing in 0.8.0 though). So we could just use that.